README.md

```diff
@@ -20,9 +20,17 @@ To run the prototype:
 1. Clone a local copy of the master branch of this repo
 2. `npm run start:firefox`
 
-This command will open a new firefox window which has the extension installed. The module hot-refreshes the browser when there is a change in the source code. To force an update in the browser, simply press `r` in the terminal where web-ext is running.
+This command will open a new firefox window which has the extension installed. However the default settings will not persist any login data, and you will need to sign in to twitter each time you stop running the command. 
 
-Currently only firefox is configured. 
+To persist your twitter login data after stopping the process, follow these instructions:
+* Create a new profile on firefox (at `about:profiles`)
+* Open an instance of firefox as this profile
+  - `firefox -p your_profile_name`
```

Comment:
You have to close firefox for this to work. Or you can do `firefox --new-instance -p your_profile_name`.

You can leave this as you like if you don't want to advertise that way if you think it's more complicated.

Reply:
Makes sense, thanks!
Will add the new instance bit.

Reply:
Done :)